transport/rif_comms/proto/api_pb2_grpc.py

```diff
@@ -2,7 +2,7 @@
 """Client and server classes corresponding to protobuf-defined services."""
 import grpc
 
-from transport.rif_comms.proto import api_pb2 as api__pb2
+import api_pb2 as api__pb2
```

Comment:
this should be rollbacked

Reply:
> this should be rollbacked

why?

Reply:
in my case, lumino didnt started because it canot find that package.

Reply:
you're right, fixed.

thanks